src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -22,6 +24,12 @@ export const PartyMap: React.FC<PartyMapProps> = ({ venue }) => {
   const { user, profile } = useUser();
   const { recentVenueUsers } = useRecentVenueUsers();
 
+  const { relatedVenueIds } = useRelatedVenues({ currentVenueId: venue.id });
+
+  const venueIds = useMemo(() => relatedVenueIds, [relatedVenueIds]);
```

Comment:
`relatedVenueIds` from `useRelatedVenues` is already `useMemo`'d, so you don't need to do it again here.

Also, this `useMemo` as written is a noop that will waste resources, as it is just re-memoing the value whenever the value changes, which has no use/purpose.

Reply:
fixed